### PR TITLE
Adding gif and tape scilla

### LINKED ISSUE #5: Update GIF for scilla
https://github.com/edoardottt/scilla

Use [`charmbracelet/vhs`](https://github.com/charmbracelet/vhs) to customize the GIF. 

- The GIF must reproduce the exact same commands you can see on the GIF now
- Be sure to add here both the .tape and the GIF file.

Let me know if you need some guidance :)

### PR DESCRIPTION
**Pull request by PaolaDaft**
Hey, issue #5 is done c;

### CODE REVIEW

File: scilla/scilla.tape [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# Display:
+#   Hide                            Hide the subsequent commands from the output
+#   Show                            Show the subsequent commands in the output
+
+Output examples/scilla.gif
+
+Require echo
+
+Set Shell "bash"
+Set FontSize 32
```
Comment (edoardottt) [inline scilla/scilla.tape]: imo the font size is too high... what if we try with something like 17 ? let's see how it goes

File: scilla/scilla.tape [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+Set Width 1200
+Set Height 600
+
+Type "scilla subdomain -target google.com" Sleep 500ms  Enter
+
+Sleep 5s
```
Comment (edoardottt) [inline scilla/scilla.tape]: Maybe we should sleep a lil bit longer here, like 8s

**Review by edoardottt — CHANGES_REQUESTED** [review]

File: scilla/scilla.tape [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+Output examples/scilla.gif
+
+Require echo
+
+Set Shell "bash"
+Set FontSize 32
+Set Width 1200
+Set Height 600
+
+Type "scilla subdomain -target google.com" Sleep 500ms  Enter
```
Comment (edoardottt) [inline scilla/scilla.tape]: sleep a lil bit longer, like 1500ms

**Review by edoardottt — CHANGES_REQUESTED** [review]

**Comment by PaolaDaft** [comment]
I already made the changes you requested, I also modified the size, let me know if you want another change 👀

File: scilla/scilla.tape [hunk 4]
```diff
@@ -52,15 +52,15 @@
...
-Output examples/scilla.gif
+Output scilla.gif
 
 Require echo
 
 Set Shell "bash"
-Set FontSize 32
-Set Width 1200
-Set Height 600
+Set FontSize 17
```
Comment (edoardottt) [inline scilla/scilla.tape]: Let's try with `16`

File: scilla/scilla.tape [hunk 5]
```diff
@@ -52,15 +52,15 @@
...
+Set Width 1920
```
Comment (edoardottt) [inline scilla/scilla.tape]: Width `1200` is okay

File: scilla/scilla.tape [hunk 6]
```diff
@@ -52,15 +52,15 @@
...
+Set Height 944
```
Comment (edoardottt) [inline scilla/scilla.tape]: Height `600` is okay

File: scilla/scilla.tape [hunk 7]
```diff
@@ -52,15 +52,15 @@
...
 
-Type "scilla subdomain -target google.com" Sleep 500ms  Enter
+Type "scilla subdomain -target google.com" Sleep 1500ms  Enter
 
-Sleep 5s
+Sleep 8s
```
Comment (edoardottt) [inline scilla/scilla.tape]: Sleep for a bit longer, like `10s`

**Review by edoardottt — CHANGES_REQUESTED** [review]
Requesting small changes

**Comment by PaolaDaft** [comment]
Done, something else? 👀

**Review by edoardottt — APPROVED** [review]
lgtm

**Comment by edoardottt** [comment]
Do you know why at a certain point the enumeration stops? :/

**Comment by PaolaDaft** [comment]
From what I see, it's because the whole numbering takes about 4 minutes, and neither this gif nor the one you had before completes the numbering ;-;

**Comment by PaolaDaft** [comment]
Oh, and it also has to do with the network, if the numbering is slower or faster. I tried with a 40s sleep and the numbering reaches 6%, on my machine

**Comment by edoardottt** [comment]
I don't mean to complete the enumeration, 100% would be too much to show. But at least some seconds going forward without stopping itself. Have you tried to produce the gif multiple times? Does it always stop?

**Comment by PaolaDaft** [comment]
Yes, if I have tried to produce the gif several times, in most of them it stops, the one that has not stopped so much is this one:
scilla4

**Review by edoardottt — APPROVED** [review]
lgtm

**Pull request merged**